src/lib.rs

```diff
@@ -420,6 +424,18 @@ impl Config {
         self
     }
 
+    /// Disables the generation of default compiler cxx flags. The default compiler flags may cause conflicts in some cross compiling scenarios.
```

Comment:
You've mixed up the docstrings.

Anyways, there must be a cleaner way without the duplication. What if the same applies for other language targets down the line (Fortran, ASM)

Reply:
What's the meaning of `cleaner way without the duplication`. I think it's necessary to clear the default flags by languages instead of simply cleaning them for all the language targets

Reply:
Come on, half the code is `s/cxx/c`. At least use something like a mapping `default_flags` that can hold the option for different langs.